sdk/keyvault/azure-security-keyvault-administration/src/test/java/com/azure/security/keyvault/administration/KeyVaultBackupClientTest.java

```diff
@@ -42,14 +42,44 @@ public void beginBackup(HttpClient httpClient) {
 
         SyncPoller<KeyVaultBackupOperation, String> backupPoller = client.beginBackup(blobStorageUrl, sasToken);
 
-        PollResponse<KeyVaultBackupOperation> backupResponse = backupPoller.waitForCompletion();
+        backupPoller.waitForCompletion();
 
         String backupBlobUri = backupPoller.getFinalResult();
 
         assertNotNull(backupBlobUri);
         assertTrue(backupBlobUri.startsWith(blobStorageUrl));
     }
 
+    /**
+     * Tests that a backup operation can be obtained by using its {@code jobId}.
+     */
+    @ParameterizedTest(name = DISPLAY_NAME)
+    @MethodSource("com.azure.security.keyvault.administration.KeyVaultAdministrationClientTestBase#createHttpClients")
+    public void getBackupStatus(HttpClient httpClient) {
+        if (getTestMode() != TestMode.PLAYBACK) {
+            // Currently there is no Managed HSM environment for pipeline testing.
+            // TODO: Remove once there is a proper cloud environment available.
+            return;
+        }
+
+        client = getClientBuilder(httpClient, false).buildClient();
+
+        SyncPoller<KeyVaultBackupOperation, String> backupPoller = client.beginBackup(blobStorageUrl, sasToken);
+        String jobId = backupPoller.poll().getValue().getJobId();
+
+        SyncPoller<KeyVaultBackupOperation, String> backupStatusPoller = client.getBackupOperation(jobId);
+
+        KeyVaultBackupOperation backupOperation = backupPoller.waitForCompletion().getValue();
```

Comment:
assuming this is blocking, should we let the `backupStatusPoller` do the initial polling?

Reply:
I don't think it would change much since we need to get the final result from both pollers. Maybe I don't understand all the implications when it comes to the async aspect of it.